frontend/src/hooks/useClearSWRCache.test.ts

```diff
@@ -50,4 +61,16 @@ describe('manageCacheEntries', () => {
         expect(cacheMock.has('other-2')).toBe(true);
         expect(cacheMock.has('prefix-3')).toBe(true);
     });
+
+    it('treats a negative cache size as zero', () => {
+        const cacheMock = new Map();
+        cacheMock.set('prefix-1', {});
```

Comment:
is the order of prefixes supposed to be alphabetically sorted? the way those tests are written indicates so even though it shouldn't be the case